modules/tinymce/src/plugins/pagebreak/test/ts/browser/PageBreakSanityTest.ts

```diff
@@ -1,40 +1,35 @@
-import { ApproxStructure, Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import { ApproxStructure } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinyUiActions } from '@ephox/mcagar';
 
-import PageBreakPlugin from 'tinymce/plugins/pagebreak/Plugin';
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/pagebreak/Plugin';
 import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.pagebreak.PageBreakSanityTest', (success, failure) => {
-
-  Theme();
-  PageBreakPlugin();
-
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyUi = TinyUi(editor);
-    const tinyApis = TinyApis(editor);
-
-    Pipeline.async({}, Log.steps('TBA', 'PageBreak: Click on the pagebreak toolbar button and assert pagebreak is inserted', [
-      tinyUi.sClickOnToolbar('click on pagebreak button', 'button[aria-label="Page break"]'),
-      tinyApis.sAssertContentStructure(ApproxStructure.build((s, str, arr) => {
-        return s.element('body', {
-          children: [
-            s.element('p', {
-              children: [
-                s.element('img', {
-                  classes: [
-                    arr.has('mce-pagebreak')
-                  ]
-                })
-              ]
-            })
-          ]
-        });
-      }))
-    ]), onSuccess, onFailure);
-  }, {
+describe('browser.tinymce.plugins.pagebreak.PageBreakSanityTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
     plugins: 'pagebreak',
     toolbar: 'pagebreak',
     base_url: '/project/tinymce/js/tinymce'
-  }, success, failure);
+  }, [ Theme, Plugin ]);
+
+  it('TBA: PageBreak: Click on the pagebreak toolbar button and assert pagebreak is inserted', () => {
```

Comment:
```suggestion
  it('TBA: Click on the pagebreak toolbar button and assert pagebreak is inserted', () => {
```
Need to remove the plugin name here.